server/infrastructure/portal/mapper/PortalDataMapper.ts

```
@@ -0,0 +1,150 @@
+        semester: parseInt(semester.semester.split('-')[1]),
+        courses: semester.courses.map(course => ({
+          code: course.subjtCd,
+          name: course.subjtNm,
+          professor: course.ltrPrfsNm ?? '미확인 교수',
+          department: course.estbDpmjNm,
+          credits: course.point,
+          grade: course.cretGrdCd,
+          isRetake: course.refacYearSmr !== '-' ? true : false,
+          schedule: course.timtSmryCn,
+          areaType: course.facDvnm,
+          areaCode: course.cltTerrNm ? parseInt(course.cltTerrNm) + '' : undefined,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)